.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Context
+
+(Is your feature request related to a problem?)  
+*Ex: I get frustrated when having to use nested `if` conditions where a `elif` could be used*
+
+# Description of the feature
+
+(What do you want to see happening)  
+*Ex: I would like the keyword `elif` to be added to Inmanta Language*
+
+# Describe alternatives you've considered
+
+(Have you considered doing without this feature? How?)  
+*Ex: The same behavior can be achieved currently doing something like:*
+```
+if condition1:
+    ...
+else:
+    if contition2:
+    ...
+    end
+end
+```
+
+# (Optional) Are you willing to contribute and try implementing this feature?
```

Comment:
I would remove this section.

Reply:
Removed it in https://github.com/inmanta/inmanta/pull/2416/commits/2e2f1f6a155d159d30daa2953c86835f4bde579f